api/test/unit/utils/schedulerUtils.test.js

```diff
@@ -31,11 +32,12 @@ describe('Scheduler Utility functions', function() {
   context ('Creating schedules during policy creation', function () {
 	  it('should create new schedules for app id 12345',function(done){
 		    nock(schedulerURI)
-		    .put('/v2/schedules/12345')
+		    .put(/\/v2\/schedules\/12345\?guid=.*/)
 		    .reply(204);
 		    var mockRequest = {
 		        body : fakePolicy,
-		        params : { 'app_id' : '12345' }
+		        params : { 'app_id' : '12345' },
+		        query : {'policy_guid' : uuidV4() }
```

Comment:
I was wondering if you specify fixed value to 'policy_guid', you could specify fixed value to 'guid' at line 35 same as 'app_id'.